### DISCUSSION TITLE
Collapse 1.80.x Feedback Forum

### DISCUSSION THREAD

**Discussion by bagusnl** [neon-nyan/Collapse#451] [General]
Hi~
In the new 1.80.x update we have changed quite a lot of the UI/UX for Collapse and also added some improvements and fixes here and there.

Please let us know how you feel about this update!

**Comment by alvinvin00** [comment]
![image](https://github.com/CollapseLauncher/Collapse/assets/44573205/bee41745-dce2-4c0a-b6d6-fd14cc40abee)
the game's logo in here are too close to topbar and sidebar, need more margin on top left side like 16px or so

**Reply by pha1n0q** [reply]
Looks even weirder with the small window configuration lol
![image](https://github.com/CollapseLauncher/Collapse/assets/88598340/43aca71e-bf17-4f22-a83f-33f8de25598f)

**Reply by bagusnl** [reply]
The logo itself is hard embedded with the background provided officially. We had other plan to regarding the side bar. 
Initially we were gonna make the left bar transparent but it ended up making the sidebar buttons hard to see on certain background color, other idea is to extend the window horizontally to keep the whole background at 16:9 and add the sidebar

**Reply by shatyuka** [reply]
You should clear the image cache, as it says in the changelog.
![](https://github.com/CollapseLauncher/Collapse/assets/31368738/4b41a0ac-4a3d-4397-9908-dfc2690ebc04)

Also maybe we can make the sidebar bg transparent as default and change the color on mouse hover.

**Reply by bagusnl** [reply]
> Also maybe we can make the sidebar bg transparent as default and change the color on mouse hover.

I think this is a nice addition and we can try it, what do you think @neon-nyan ?

**Reply by neon-nyan** [reply]
Adding a gradient/solid semi-transparent background while hovering it would be nice. But we need to find out how to make a shadow behind those buttons to make it visible to the eyes while it's not hovered. Pretty much the transition between non-hovered and hovered state will be done in app code instead of XAML (I've had enough with it lmao)

**Reply by shatyuka** [reply]
> Also maybe we can make the sidebar bg transparent as default and change the color on mouse hover.

Done by b8156af and c3bd6a4.

**Comment by Vermilion-Sinsha** [comment]
on OOBE page, can select a video as a custom background, but if actually select a video, launcher crashes.

**Reply by neon-nyan** [reply]
This caused by a video frame not being exist in the OOBE page yet.
We will be adding it in the next preview version as for now it's not available in 1.80.0-preview.

**Comment by IhoFox** [comment]
looks like the playtime box alignment is fix at the right side now, so high numbers end up taking space from the start button
![Start_Time](https://github.com/CollapseLauncher/Collapse/assets/133899447/78fe2d26-6762-4b15-b6db-b38a78b8855a)

**Reply by shatyuka** [reply]
I think we can remove the notification icon in the playtime button, it attracts too much attention. @bagusnl

**Reply by bagusnl** [reply]
> I think we can remove the notification icon in the playtime button, it attracts too much attention. @bagusnl

I believe we had a conversation with @gablm and @Cryotechnic about it where we needed something to indicate the "Last opened in" feature without interfering too much in the UX side of thing and the icon is chosen

**Reply by shatyuka** [reply]
Fixed in commit fb5ae1f

**Comment by bagusnl** [comment]
List of known issues:

## Metadata and Runtime Related:
1. Game Category/Region can't be selected after a metadata update
2. Metadata update causes background to go paimon mode/errors

(add more in comments)

**Reply by neon-nyan** [reply]
## Update: Metadata and Runtime Related:

- [x] ~~Game Category/Region can't be selected after a metadata update~~ **(Fixed on both our internal branch and public main branch)**
- [x] ~~Metadata update causes background to go paimon mode/errors~~ **(Fixed on both our internal branch and public main branch)**

**Comment by zakidze** [comment]
the  game have arrived to 90% donwlaod  the laucnher resterted it self  and updated :) now the game donwload have been back from 0% :) :)

reverted to the old launcher now i m continuing from where it was

**Comment by Deztract** [comment]
I have question, is there a way to make launcher also launch other application before game launch?

**Reply by bagusnl** [reply]
Yes, go to game settings section and scroll to the very bottom, you will see Advanced Settings and there lives the pre and post game launch commands

**Reply by Deztract** [reply]
> Yes, go to game settings section and scroll to the very bottom, you will see Advanced Settings and there lives the pre and post game launch commands

Ye, I found it, but I duno what commands to use for this, I tried to find any info on this, but without success, I don't know much on this topic, so sorry for bothering you

**Reply by bagusnl** [reply]
It is intentionally not very well documented because it's a a very advanced feature. But practically, it's kinda like a non-interactable command prompt that you can launch whatever before and/or after game launched

**Reply by Deztract** [reply]
> It is intentionally not very well documented because it's a a very advanced feature. But practically, it's kinda like a non-interactable command prompt that you can launch whatever before and/or after game launched

My main problem is what command I should use, like is there should be something like "run [file_path]" or something similar to this. Me dumb

**Reply by bagusnl** [reply]
`cd /d (path of your program to run) && (full path of your program)`
E.g. 
`cd /d "D:\test app" && "D:\test app\run.exe" `

Come to think of it, I should make a very simple guides for it LMAO

**Reply by Deztract** [reply]
> `cd /d (path of your program to run) && (full path of your program)` E.g. `cd /d "D:\test app" && "D:\test app\run.exe" `
> 
> Come to think of it, I should make a very simple guides for it LMAO

Oh, that worked, thank you!
